ports/servoshell/minibrowser.rs

```diff
@@ -56,18 +64,62 @@ impl Minibrowser {
             .egui_ctx
             .set_pixels_per_point(window.hidpi_factor().get());
 
+        let widget_surface_fbo = match webrender_surfman.context_surface_info() {
+            Ok(Some(info)) => info.framebuffer_object,
+            Ok(None) => panic!("Failed to get widget surface info from surfman!"),
+            Err(error) => panic!(
+                "Failed to get widget surface info from surfman! {:?}",
+                error
+            ),
```

Comment:
```suggestion
                "Failed to get widget surface info from surfman! {error:?}",
            ),
```

Reply:
done in f96a28523112620bdee12fd2b88399e33f47ba0a, though i had to bump servoshell to rust 2021